application/models/Question.php

```diff
@@ -176,10 +182,12 @@ public function rules()
                 $aRules[] = array('title', 'match', 'pattern' => '/comment$/', 'not' => true, 'message' => gT("'comment' suffix can not be used with multiple choice with comments."));
             }
         } else {
-            // Disallow other if sub question have 'other' for title
-            $oSubquestionOther = Question::model()->find("parent_qid=:parent_qid and LOWER(title)='other'", array("parent_qid" => $this->qid));
-            if ($oSubquestionOther) {
-                $aRules[] = array('other', 'compare', 'compareValue' => 'Y', 'operator' => '!=', 'message' => sprintf(gT("'%s' can not be used if the 'Other' option for this question is activated."), 'other'), 'except' => 'archiveimport');
+            if ($this->getAsSubquestions()) {
+                // Disallow other if sub question have 'other' for title
+                $oSubquestionOther = Question::model()->find("parent_qid=:parent_qid and LOWER(title)='other'", array("parent_qid" => $this->qid));
```

Comment:
Maybe we can wrap this into a method?

Reply:
you mean something like `$oSubquestionOther = $this->getAsOtherSubquestion()` ?

Can be better, right.

Reply:
why are there methods names like `getAsxxxx`?
What do getAsxxx means?

Reply:
`hasSubquestionsOther` ?
Do we need the `get` prefix?

Reply:
`$question->hasubquestionother` == `$question->getHasOtherSubquestion()` no ?

Reply:
https://github.com/LimeSurvey/LimeSurvey/pull/2443/files#diff-75d3428e7aa3f16b9dc93ef54dfa4d0556c04431cb53510749ded531d549a148R1488

Reply:
You could have like this 
```
function getSubquestionOther() {
   return Question::model()->find("parent_qid=:parent_qid and LOWER(title)='other'", array("parent_qid" => $this->qid));
}
```

then use `getSubquestionOther()` in the code.